Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -178,6 +179,9 @@ -(void)applySubviewClipsToBoundsHack {
 
 - (void)setup {
     self.gregorian = [[NSCalendar alloc] initWithCalendarIdentifier:NSCalendarIdentifierGregorian];
+    self.dateFormatter = [NSDateFormatter new];
+    self.dateFormatter.locale = locale;
```

Comment:
This doesn't compile. We need to use `self.locale`. We also need to update this property in `setLocale:`